### DISCUSSION TITLE
[Q&A] progress??

### DISCUSSION THREAD

**Discussion by bachig26** [earthtraveller1/tetris-cli#5] [Q&A]
1. When's the 1st useable version (like ready to use version) will be available?

2. can you able to share some screenshots or clips?

3. ....

**Comment by earthtraveller1** [comment]
1. I'm not sure since I don't exactly have a clear roadmap for this project.
2. No because since I'm still working on the back-ends, there's not really much to show yet.

**Comment by nli33** [comment]
Please be aware of the following repository: https://github.com/carpetmaker3162/cli-tetris

Thank you, and glory to the People's Republic of China!

**Comment by earthtraveller1** [comment]
Also, for further context, this repository is intended to be a Rust remake of the original project by @carpetmaker3162 that I started for no reasons.
